cmd/cosign/cli/attest/attest_blob.go

```diff
@@ -126,6 +143,52 @@ func (c *AttestBlobCommand) Exec(ctx context.Context, artifactPath string) error
 		return errors.Wrap(err, "signing")
 	}
 
+	signedPayload := cosign.LocalSignedPayload{}
+	if c.TSAServerURL != "" {
+		clientTSA, err := tsaclient.GetTimestampClient(c.TSAServerURL)
+		if err != nil {
+			return fmt.Errorf("failed to create TSA client: %w", err)
+		}
+		rfc3161Timestamp, err := tsa.GetTimestampedSignature(sig, clientTSA)
+		if err != nil {
+			return err
+		}
+		if err := os.WriteFile(c.RFC3161TimestampPath, rfc3161Timestamp, 0600); err != nil {
```

Comment:
We'll need to transform the response into the expected JSON - https://github.com/sigstore/cosign/blob/631a26fb92347230b423bedfbaac84ef177a6fd0/cmd/cosign/cli/sign/sign_blob.go#L91-L104

That's what Verify expects - https://github.com/sigstore/cosign/blob/631a26fb92347230b423bedfbaac84ef177a6fd0/cmd/cosign/cli/verify/verify_blob.go#LL231C14-L231C14

Reply:
I can also be convinced that we don't need this intermediate representation, but then we should also update sign-blob/verify-blob

Reply:
ah thanks for the catch! i'll make sure it's fixed on the verify-blob-attestation path too.